_includes/menu-item-ids/table-menu-items.md

```diff
@@ -25,3 +25,13 @@
 | `tablepasterowafter`      | Table                                                                                                    | Paste row after the currently selected row {{site.requires_5_4v}}          |
 | `tablerowprops`           | Table                                                                                                    | Opens the row properties dialog                                           |
 | `tabledeleterow`          | Table                                                                                                    | Deletes the currently selected row or rows {{site.requires_5_4v}}             |
+| `tableclass`              | _Not Applicable_                                                                                         | Adds or removes pre-defined classes to the selected table. {{site.requires_5_9v}}                   |
+| `tablecellclass`          | _Not Applicable_                                                                                         | Adds or removes pre-defined classes to selected cells in the table. {{site.requires_5_9v}}          |
+| `tablecellvalign`         | _Not Applicable_                                                                                         | Sets the vertical alignment of the selected cells. {{site.requires_5_9v}}                           |
+| `tablecellborderwidth`    | _Not Applicable_                                                                                         | Sets the border width of the all selected cells. {{site.requires_5_9v}}                             |
```

Comment:
re-phrase from: Sets the border width of the all selected cells.
to:
Sets the border width to all the selected cells.

Reply:
```suggestion
| `tablecellborderwidth`    | _Not Applicable_                                                                                         | Sets the border width of the selected cells. {{site.requires_5_9v}}                             |
```

Reply:
good catch @EkimChau